package.json

```diff
@@ -52,15 +52,16 @@
   },
   "prettier": {},
   "dependencies": {
-    "retext": "^8.1.0",
-    "retext-smartypants": "^5.2.0",
+    "retext": "^9.0.0",
+    "retext-smartypants": "^6.0.0",
```

Comment:
Now that these dependencies have been updated to a new major, I have a feeling #78 needs to be resolved as well. Would you know how to include that in this, or a separate PR?

Reply:
let's do in separate PR, and maybe to convert to typescript this package

Reply:
Yep, TypeScript setup definitely. I vote for a Vite setup if you agree, seems simplest, and we already have it as a dependency.

Reply:
vite by default bundle dependencies and used mostly for front-end apps, I will vote for tsc or tsup